packages/cypress/package.json

```diff
@@ -72,11 +76,9 @@
   "dependencies": {
     "@chromaui/rrweb-snapshot": "2.0.0-alpha.17-noAbsolute",
     "@segment/analytics-node": "^1.1.0",
-    "@storybook/addon-essentials": "^8.6.0",
     "@storybook/csf": "^0.1.0",
```

Comment:
It doesn't look like we use this dependency at all. Would now be a good time to remove it?

Reply:
Good call, I'll check